tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -1184,7 +1170,7 @@ class MklFusedBatchNormGradOp : public OpKernel {
 #else
       diff_dst_data =
           static_cast<T*>(const_cast<T*>(diff_dst_tensor.flat<T>().data()));
-#endif
+#endif // ENABLE_MKLDNN_V1
```

Comment:
Nit: There are two spaces between `#endif` and `//`.
```suggestion
#endif  // ENABLE_MKLDNN_V1
```